python/grass/jupyter/seriesmap.py

```diff
@@ -12,20 +12,19 @@
 #           for details.
 """Create and display visualizations for a series of rasters."""
 
-import tempfile
 import os
-import weakref
 import shutil
 
+
 import grass.script as gs
 from grass.grassdb.data import map_exists
+from grass.jupyter.baseseriesmap import BaseSeriesMap
 
 from .map import Map
 from .region import RegionManagerForSeries
```

Comment:
```suggestion

from .map import Map
from .region import RegionManagerForSeries
from .baseseriesmap import BaseSeriesMap
```

Reply:
You will also have to add baseseriesmap to Makefile.

Reply:
Do I need to add baseseriesmap to the __init__ file also?

Reply:
No, it won't be exposed as part of the API.